src/middlewares/userValidations.js

```diff
@@ -31,8 +31,7 @@ const validateUser = path => (req, res, next) => {
  * @returns {object} JSON response
  */
 const emailExists = (req, res, next) => {
-  let { email } = req.body;
-  email = email.trim().toLowerCase();
+  const { email } = req.body;
```

Comment:
This need to be trimmed and converted to lower case.
```suggestion
  let { email } = req.body;
  email = email.trim().toLowerCase();
```